addons/payment/models/payment_provider.py

```diff
@@ -64,10 +64,12 @@ def _valid_field_parameter(self, field, name):
     redirect_form_view_id = fields.Many2one(
         string="Redirect Form Template", comodel_name='ir.ui.view',
         help="The template rendering a form submitted to redirect the user when making a payment",
+        ondelete="restrict",
```

Comment:
```suggestion
        ondelete='restrict',
```
We tend to use `"quacking for the user"` when the string can be read by the user/translated and `'technical quack'` for purely technical purpose string =]